dev/com.ibm.ws.classloading.bells/src/com/ibm/ws/classloading/bells/internal/Bell.java

```diff
@@ -329,9 +304,28 @@ private static void getServiceInfos(ArtifactEntry providerConfigFile, String ser
             try {
                 final BufferedReader reader = createReader(providerConfigFile);
                 try {
-                    String implClass;
-                    while ((implClass = reader.readLine()) != null) {
-                        serviceInfos.add(new ServiceInfo(providerConfigFile, implClass));
+                    String line;
+                    Hashtable<String, String> props = new Hashtable<String, String>();
+                    while ((line = reader.readLine()) != null) {
+                        int index = line.indexOf(COMMENT_CHAR);
+                        String implClass;
+                        if (index != -1) {
```

Comment:
Are we concerned about trailing comments, i.e.:

com.acme.MyServiceImpl   # My Service Implementation

Reply:
(Not sure that is even valid)

Reply:
Oh. I retract, looks like you covered it.

Reply:
yep, trailing comments are valid and I tried to cover it.